specification/storage/resource-manager/Microsoft.Storage/stable/2021-02-01/storage.json

```diff
@@ -2065,6 +2065,43 @@
       ],
       "description": "The custom domain assigned to this storage account. This can be set via Update."
     },
+    "SasPolicy": {
+      "properties": {
+        "sasExpirationPeriod": {
+          "type": "string",
+       	  "example": "1.15:59:59",
+          "description": "The SAS expiration period, DD.HH:MM:SS."
+        },
+        "expirationAction": {
+		  "type": "string",
+		  "enum": [
+		    "Log"
+		  ],
+		  "x-ms-enum": {
+		    "name": "ExpirationAction",
+		    "modelAsString": true
+		  },
+		  "default": "Log",
+		  "description": "The SAS expiration action. Can only be Log."
+		}
+      },
+      "required": [
+        "sasExpirationPeriod"
```

Comment:
Per my test, it seems ExpirationAction is also required in SasPolicy, or will report error  "Values for request parameters are invalid: keyPolicy.expirationAction."  (The error is also wrong)

Reply:
Make ExpirationAction as required.
SRP fix will be rolled out in next release